packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
+  display: flex;
+  flex-direction: row;
+  color: var(--grey100);
+  padding-top: 32px;
+  padding-bottom: 32px;
+  container-name: CdgAbout;
+  container-type: inline-size;
+  justify-content: space-between;
+
+  .cdg-about-left {
+    width: 50%;
```

Comment:
Parent is flex with row direction, so in here you just need to use flex: 1 for both 2 columns.

Reply:
But it not work with `flex-direction: column` for mobile